SimpleJavaProject/src/main/java/cristina_savrin/StringsMain.java

```diff
@@ -0,0 +1,32 @@
+package cristina_savrin;
+
+import java.util.Scanner;
+
+public class StringsMain {
+
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.print("Enter a string: ");
+        String string = scanner.nextLine();
+
+        // 1. Count the number of characters in a given string.
+        System.out.println("> The number of characters in string is: " + string.length());
+
+        // 2. Convert a string to uppercase.
+        System.out.println("> The string in uppercase: " + string.toUpperCase());
+
+        // 3. Check if a string contains a specific substring.
+        if (string.contains("java") || string.contains("JAVA")){
```

Comment:
please use to upper case or to lowercase 
because it is not correct to verify all cases like JaVA or jAvA

Reply:
Done.